cognite/client/_api/functions.py

```diff
@@ -175,20 +189,20 @@ def __iter__(self) -> Iterator[Function]:
 
     def create(
         self,
-        name: str,
+        name: str | FunctionWrite,
```

Comment:
Note that if you use `FunctionWrite` then you also have to pass in a `file_id`, which means that the only way to deploy a function through a `function_handle` or `function_path` is to have this as `name`.